fake.js

```diff
@@ -138,6 +138,9 @@ var randomElement = function(array) {
   return array[Math.floor(Math.random() * array.length)];
 };
 
+var generateEmailAddress = function (username, domain) {
+    return (name + '@' + domain).toLowerCase();
```

Comment:
Yes I did, noticed the mistake myself and amended my commit. But for some reason the push timed out. Check again.